public/components/event_analytics/explorer/__tests__/data_grid.test.tsx

```diff
@@ -45,13 +45,23 @@ describe('Datagrid component', () => {
       Promise.resolve((sampleEmptyPanel as unknown) as HttpResponse)
     );
 
+    const tabId = 'explorer-tab-_fbef9141-48eb-11ee-a60a-af33302cfb3c';
+
     const http = httpClientMock;
     const pplService = new PPLService(httpClientMock);
-    const store = createStore(rootReducer, applyMiddleware(thunk));
+    const preloadedState = {
```

Comment:
Could you add tests for default time and user selected timestamp cases?

Reply:
added that extra test